packages/manager/apps/public-cloud/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@ovh-ux/manager-public-cloud",
+  "version": "2.0.0",
+  "private": true,
+  "license": "BSD-3-Clause",
+  "main": "./src/index.js",
+  "scripts": {
+    "build": "webpack",
+    "dev": "webpack-dev-server",
+    "dev:watch": "yarn run dev",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-stack' --include-filtered-dependencies -- yarn run build",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-stack' --include-filtered-dependencies -- yarn run dev",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-stack' --include-filtered-dependencies -- yarn run dev:watch"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-core": "^2.0.0",
+    "@ovh-ux/ng-ovh-api-wrappers": "^3.0.0",
+    "@ovh-ux/ng-ovh-apiv7": "^2.0.0",
+    "@ovh-ux/ng-ovh-otrs": "^7.0.0-beta.1",
+    "@ovh-ux/ng-ovh-sidebar-menu": "^8.0.0-beta.0",
+    "@uirouter/angularjs": "^1.0.20",
+    "angular": "^1.7.5",
+    "angular-resource": "^1.7.5",
+    "angular-translate": "^2.18.1",
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
+    "jquery": "^2.1.3",
+    "lodash": "^3.10.1",
+    "moment": "^2.22.2",
+    "ovh-api-services": "^4.0.0",
```

Since you declared a dependency to both`'@ovh-ux/ng-ovh-api-wrappers'` and `'@ovh-ux/ng-ovh-apiv7'` shouldn't be?:

```suggestion
    "ovh-api-services": "^6.0.0",
```

NB: `'@ovh-ux/ng-ovh-api-wrappers'` is enough. You can get rid of `'@ovh-ux/ng-ovh-apiv7'` I guess.